common-files/.mvn/extensions.xml

```diff
@@ -2,6 +2,6 @@
   <extension>
     <groupId>io.jenkins.tools.incrementals</groupId>
     <artifactId>git-changelist-maven-extension</artifactId>
-    <version>1.1</version>
+    <version>1.2-SNAPSHOT</version> <!-- TODO https://github.com/jenkinsci/incrementals-tools/pull/16 -->
```

Comment:
Cannot prepare a timestamped snapshot, so waiting for upstream to be approved to merge and release.